canonical-web-and-design/ubuntu.com
# Build new kubernetes managed page

## Done

- Rebuild `/managed/kubernetes`
- Removed the local p-separator file as the vanilla one is the same.

## QA

- View page at: https://ubuntu-com-10550.demos.haus/kubernetes/managed
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the page against the [design](https://github.com/canonical-web-and-design/ubuntu.com/issues/10470#issuecomment-931418091) and [copy doc](https://docs.google.com/document/d/1Qv379nYnod9e4jMrsdDND8D7mbPyqELi2lowjkfD-gU/edit#) and resolve all comments
- Resources section is slightly different from the design as I used a h4 instead of muted heading - suggested [here](https://github.com/canonical-web-and-design/vanilla-framework/pull/3935)


## Issue / Card

Fixes #10471
